BitwardenShared/Core/Vault/Services/TestHelpers/MockFolderService.swift

```diff
@@ -5,6 +5,7 @@ import Combine
 
 class MockFolderService: FolderService {
     var addedFolderName: String?
+    var addFolderWithServerFolder = Folder.fixture()
```

Comment:
🎨 I believe this should be changed to a Result so errors can be thrown:
```suggestion
    var addFolderWithServerResult: Result<Folder, Error> = .success(Folder.fixture())
```